tests/reconfig/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,87 @@
+
+### Event Batch Processing
+
+| Test number | NumResources | Event Batch Total | Event Batch Processing avg time (ms) | <= 500ms | <= 1000ms | <= 5000ms | <= 10000ms | <= 30000ms |
+|-------------|--------------|-------------------|--------------------------------------|----------|-----------|-----------|------------|------------|
+| 1           | 30           | 5                 | 733.6                                | 80%      | 80%       | 100%      | 100%       | 100%       |
+| 1           | 150          | 5                 | 2967                                 | 40%      | 40%       | 40%       | 40%        | 40%        |
+| 2           | 30           | 371               | 57.32                                | 100%     | 100%      | 100%      | 100%       | 100%       |
+| 2           | 150          | 1743              | 75.87                                | 98.45%   | 100%      | 100%      | 100%       | 100%       |
+| 3           | 30           | 370               | 37.48                                | 99.73%   | 99.73%    | 100%      | 100%       | 100%       |
+| 3           | 150          | 1808              | 40.18                                | 99.94%   | 99.94%    | 99.94%    | 99.94%     | 100%       |
+
+## NumResources to Total Resources
+
+| NumResources | Gateways | Secrets | ReferenceGrants | Namespaces | application Pods | application Services | HTTPRoutes | Attached HTTPRoutes | Total Resources |
+|--------------|----------|---------|-----------------|------------|------------------|----------------------|------------|---------------------|-----------------|
+| x            | 1        | 1       | 1               | x+1        | 2x               | 2x                   | 3x         | 2x                  | <total>         |
+| 30           | 1        | 1       | 1               | 31         | 60               | 60                   | 90         | 60                  | 244             |
+| 150          | 1        | 1       | 1               | 151        | 300              | 300                  | 450        | 300                 | 1204            |
+
+> Note: Only 2x HTTPRoutes attach to the Gateway because the parentRef name in the `cafe-tls-redirect` HTTPRoute is incorrect. This has been fixed, but until https://github.com/nginxinc/nginx-gateway-fabric/issues/1107 is fixed we can't actually run the test successfully.
+
+## Observations
+
+1. Reload time seems to have a increased slightly in a few instances, though time to ready is consistent if not faster.
+
+2. We appear to be processing fewer batches, especially for Test 1 where the resources exist before NGF. This subsequently increased the time to process each
+batch since each batch was much larger. I don't think we changed any of this logic recently, so not really sure why it's happening. Test 1 number seems very different
```

Comment:
probably because of this https://github.com/nginxinc/nginx-gateway-fabric/commit/e738ab0575024acff56bfb250a6ddb9181efc12b. 
less changes propagate into the event loop. As a result, less number of batches

Reply:
Ah, you're probably right.